clusterloader2/pkg/measurement/common/network/network_performance_measurement.go

```diff
@@ -0,0 +1,506 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package network captures network performance metrics
+// for protocol TCP,UDP,HTTP etc. The metrics are collected for baseline (1:1),
+// scale (N:M) pod ratios.Client and server pods located on different worker
+// nodes exchange traffic for specified time to measure the performance metrics.
+package network
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"k8s.io/client-go/dynamic"
+
+	"k8s.io/client-go/tools/cache"
```

Comment:
please move those imports below (to sort them alphabetically)

Reply:
Done